holoviews/ipython/__init__.py

```diff
@@ -254,6 +191,24 @@ def _get_resources(self, args, params):
             resources = ['holoviews'] + resources
         return resources
 
+    @classmethod
+    def load_hvjs(cls, logo=False, JS=True, message='HoloViewsJS successfully loaded.'):
```

Comment:
Just wondering if anything else might have been using it. Why wasn't it a classmethod to begin with?